MockKit/UnfinalizedDose.swift

```diff
@@ -125,11 +132,18 @@ public struct UnfinalizedDose: RawRepresentable, Equatable, CustomStringConverti
     }
 
     public var description: String {
+        let unitsStr = insulinFormatter.string(from: NSNumber(value:units)) ?? "?"
         switch doseType {
         case .bolus:
-            return "Bolus units:\(units) startTime:\(startTime) duration:\(String(describing: duration))"
+            if let scheduledUnits = scheduledUnits,
```

Comment:
[nit] similarly here: it would be great to bring this out into a utility and bring it under unit test.